### DISCUSSION TITLE
tinygrad.org going live

### DISCUSSION THREAD

**Discussion by geohot** [tinygrad/tinygrad#414] [Announcements]
repo is geohot/tinyxxx

Looking for people to write good documentation.

**Comment by ofou** [comment]
do you want Sphinx docs, right? (like PyTorch)

**Comment by etherx-dev** [comment]
Nice

**Comment by yedidmeloul** [comment]
hello everyone im on github but do not understand what is it exactly and for what is it use
someone help
